### ISSUE TITLE
Fragments not working

### ISSUE THREAD

**Issue by basilfx** [eamigo86/graphene-django-extras#1]
First of all: thank you for contributing this! Django is great, GraphQL is great, but Relay was too much for me.

I noticed that queries with fragments are not yet supported. For instance:

```graphql
query Objects {
  object {
    results(offset: 0, limit: 20) {
      ...ObjectFragment
    }
    totalCount
  }
}

fragment ObjectFragment on ObjectType {
  id
  __typename
}
```

I have worked around the issue by changing `utils.py` on line 105, since it looks like that method only optimizes the query:

```python
...
elif hasattr(field, 'selection_set') and field.selection_set:
...
```

I found something that might help [here](https://github.com/graphql-python/graphene/issues/378#issuecomment-264048418). See the `get_fields` method.

Here is a traceback:

```
Traceback (most recent call last):
  File "/env/lib/python3.6/site-packages/graphql/execution/executor.py", line 191, in resolve_or_error
    return executor.execute(resolve_fn, source, info, **args)
  File "/env/lib/python3.6/site-packages/graphql/execution/executors/sync.py", line 7, in execute
    return fn(*args, **kwargs)
  File "/env/lib/python3.6/site-packages/graphene_django/debug/middleware.py", line 56, in resolve
    promise = next(root, info, **args)
  File "/env/lib/python3.6/site-packages/graphql/execution/middleware.py", line 57, in make_it_promise
    return Promise.resolve(next(*a, **b))
  File "/env/lib/python3.6/site-packages/graphene_django_extras/fields.py", line 229, in list_resolver
    qs = queryset_factory(manager, info.field_asts, filtering_args, **kwargs)
  File "/env/lib/python3.6/site-packages/graphene_django_extras/utils.py", line 131, in queryset_factory
    select_related, prefetch_related = recursive_params(fields_asts[0].selection_set, available_related_fields, select_related, prefetch_related)
  File "/env/lib/python3.6/site-packages/graphene_django_extras/utils.py", line 106, in recursive_params
    a, b = recursive_params(field.selection_set, available_related_fields, select_related, prefetch_related)
  File "/env/lib/python3.6/site-packages/graphene_django_extras/utils.py", line 105, in recursive_params
    elif field.selection_set:
AttributeError: 'FragmentSpread' object has no attribute 'selection_set'
```

**Comment by eamigo86** [comment]
Sorry it was an error of mine, in that specific function of **utils.py** file I only focused on automatically optimizing and calculating the **select_related** and **prefetch_related** for each model's queryset, according to the query requested by the user, and I never took into account **fragments**. I will work on this to support fragments as soon as possible, thanks for reporting it. :thumbsup:

**Comment by eamigo86** [comment]
I just fixed the bug and it will be available in the next version (0.0.1b9) soon. 

![image](https://user-images.githubusercontent.com/11740026/31096352-6385f3d4-a781-11e7-901c-7b3ccc779378.png)

Thanks for your help :trophy:
